ArduPlane/quadplane.cpp

```diff
@@ -1423,12 +1417,12 @@ void SLT_Transition::update()
         // the quad should provide some assistance to the plane
         quadplane.assisted_flight = true;
         // update transition state for vehicles using airspeed wait
-        if (transition_state != TRANSITION_AIRSPEED_WAIT) {
+        if (transition_state != TRANSITION_AIRSPEED_WAIT && transition_state != TRANSITION_TIMER) {
```

Comment:
Everything else looks OK, I think this line is the only one that might catch us out. This is a change in behaviour, very hard to say how much it will affect different vehicles. 

I tend to think its not a good idea. I wonder it we can work around the re-triggering in a different way. Maybe a bool that is set when the force completion is triggered and cleared on transition complete.

Reply:
okay...how is this?